adventure.js

```diff
@@ -0,0 +1,17 @@
+name = prompt("What is your name, Private?");
+
+if (name) {
+  prompt ("Does it look like I care what you're name is, Private? Y/N/M")
+}
+else if (Y) {
+  prompt ("Let me show you how much I care. Push sand till I get tired!! Am I tired yet? Y/N/M")
+}
+else if (N) {
+  prompt ("Let me show you how much I care. Push sand till I get tired!! Am I tired yet? Y/N/M")
+}
+else if (M) {
+  prompt ("Let me show you how much I care. Push sand till I get tired!! Am I tired yet? Y/N/M")
+}
+if (N) {
+  prompt ("Smart answer, What was your ASVAB score? 0-31/32-70/71-99")
+}
```

Comment:
There's a problem with your syntax. `else if (Y)` is checking for a variable called `Y`, whereas you'll want to check whether the _value_ of a variable is `"Y"`. Something like this would work (it's not very DRY):

```
name = prompt("What is your name, Private?");

if (name) {
  var careAnswer = prompt ("Does it look like I care what you're name is, Private? Y/N/M")
  if (careAnswer === "Y") {
    var tiredAnswer = prompt ("Yes?! Let me show you how much I care. Push sand till I get tired!! Am I tired yet? Y/N/M")
    if (tiredAnswer === "N") {
	  var score = prompt ("Smart answer, What was your ASVAB score? 0-31/32-70/71-99")
	}
  }
  else if (careAnswer === "N") {
    var tiredAnswer = prompt ("No?! Let me show you how much I care. Push sand till I get tired!! Am I tired yet? Y/N/M")
    if (tiredAnswer === "N") {
	  var score = prompt ("Smart answer, What was your ASVAB score? 0-31/32-70/71-99")
	}
  }
  else if (careAnswer === "M") {
    var tiredAnswer = prompt ("Maybe?! Let me show you how much I care. Push sand till I get tired!! Am I tired yet? Y/N/M")
    if (tiredAnswer === "N") {
	  var score = prompt ("Smart answer, What was your ASVAB score? 0-31/32-70/71-99")
	}
  }
}
```